telosnetwork/telos-wallet
# Token balance api integration

# Fixes #179

## Description
The balance page now is displaying the actual account balances taken from the balances store. The same sorting algorithm that was present on the visual component now is part of the balance store internal logic.

## Test scenarios
- goto [this link](https://deploy-preview-281--wallet-staging.netlify.app/)
- login on EVM
  - you should see a spinner while the balances complete loading
  - you should see the list of your real balances sorted by fiat balance first and then by token balance.
  - when finished loading, you should not see the spinner
  - The TLOS token should show the token price when it is ready
- dropdown the menu of any of them
- select "send" option
  - you should be redirected to the send page with that token already selected to send

## Screenshots
![image](https://user-images.githubusercontent.com/4420760/235551028-5be1131b-1181-4381-8a9a-7eeb66104ad0.png)

![image](https://user-images.githubusercontent.com/4420760/235551092-49388f1a-c26e-4c8b-99b2-71c74994735a.png)


## Checklist:
